icu4c/source/i18n/decNumberLocal.h

```diff
@@ -259,7 +259,7 @@
   /* 2,000,000,000 (as is needed for negative exponents of            */
   /* subnormals).  The unsigned integer pow is used as a temporary    */
   /* variable. */
-  #define TODIGIT(u, cut, c, pow) {       \
+  #define TODIGIT(u, cut, c, pow) UPRV_BLOCK_MACRO_BEGIN { \
```

Comment:
Optional: This one might interfere with syncing with the upstream decnumber library. I know we already add copyright headers; do you have a sense of how much else we have patched?

I wonder if we should, in this case,
- use literal "do" and "while (0)"
- submit an upstream ticket or patch

Reply:
I don't know. The fact that the file has a perfectly normal ICU path name made me just assume that it is to be treated like any other ICU source file. If it ought to be treated differently, I would have expected it to be located in some kind of separate location in order to clearly delineate that it should be treated differently from the rest of the ICU source code.

Reply:
We keep a pristine copy of the upstream code in the "vendor" folder, and then copy into the regular ICU library location with modifications as necessary. Looking at https://github.com/unicode-org/icu/tree/master/vendor/double-conversion/upstream I don't see this file, so this change is fine.